metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/HUDocumentView.java

```diff
@@ -144,6 +154,16 @@ public int getM_HU_ID()
 		return (int)delegate.getFieldValueAsJson(I_WEBUI_HU_View.COLUMNNAME_M_HU_ID);
 	}
 
+	public I_M_HU getM_HU()
```

Comment:
if a getter like this could return `null` i think the method name should be getM_HUOrNull()